### PR TITLE
Support more maplike types

### PR DESCRIPTION
**Pull request by kornelski**

### CODE REVIEW

File: src/keyword.rs [hunk 1]
```diff
@@ -19,7 +18,7 @@ pub struct KeywordScore {
 
 impl KeywordScore {
     /// Creates a vector of `KeywordScore` from `mp`
-    pub fn from_map(mp: HashMap<String, f64>) -> Vec<Self> {
+    pub fn from_map(mp: impl IntoIterator<Item = (String, f64)>) -> Vec<Self> {
```
Comment (yaa110) [inline src/keyword.rs:21]: I think it's better to implement `std::convert::From<T: IntoIterator>` trait for `KeywordScore`. what is your idea?

**Review by yaa110 — CHANGES_REQUESTED** [review]

File: src/keyword.rs (hunk 1 above)
Comment (kornelski) [reply src/keyword.rs:21]: That would be fine too. I just kept this one for backwards compatibility.

Comment (yaa110) [reply src/keyword.rs:21]: you are right, however, since this library is in beta state, I think we could break backwards compatibility.

**Comment by kornelski** [comment]
Implementation of `FromIterator` would be more elegant, but I think this is enough, so I won't be updating this PR.

**Comment by yaa110** [comment]
Thank you for the contribution, I merged your code and I will apply the changes to implement `std::convert::From<T: IntoIterator>` trait for `KeywordScore`

**Pull request merged**